src/tirith/core/evaluators/contained_in.py

```diff
@@ -4,7 +4,7 @@
 from tirith.utils import sort_collections
 
 # TODO: At least add __name__ as the name for the logger
```

Comment:
Hi @AkashS0510, could you also remove the rest `TODO` comments, thanks

Reply:
Apologies for missing these out earlier. I have removed it now. Thank you